stdlib/public/core/Builtin.swift

```diff
@@ -293,17 +280,19 @@ public func _onFastPath() {
 
 /// Returns `true` iff the class indicated by `theClass` uses native
 /// Swift reference-counting.
-@_versioned
-@inline(__always)
-internal func _usesNativeSwiftReferenceCounting(_ theClass: AnyClass) -> Bool {
 #if _runtime(_ObjC)
-  return swift_objc_class_usesNativeSwiftReferenceCounting(
-    unsafeAddress(of: theClass)
-  )
+// Declare it here instead of RuntimeShims.h, because we need to specify
+// the type of argument to be AnyClass. This is currently not possible
+// when using RuntimeShims.h
+@_silgen_name("swift_objc_class_usesNativeSwiftReferenceCounting")
+func _usesNativeSwiftReferenceCounting(_ theClass: AnyClass) -> Bool
 #else
+@_versioned
+@inline(__always)
+func _usesNativeSwiftReferenceCounting(_ theClass: AnyClass) -> Bool {
   return true
-#endif
 }
+#endif
 
```

Comment:
@gribozavr Is it what you meant? Specially, is it correct to put `@_version` just before the non `_ObjC` version?